pkg/asset/manifests/aws/cloudproviderconfig.go

```diff
@@ -0,0 +1,75 @@
+package aws
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/pkg/errors"
+	ini "gopkg.in/ini.v1"
+
+	awstypes "github.com/openshift/installer/pkg/types/aws"
+)
+
+// CloudConfig is the aws cloud provider config.
+type CloudConfig struct {
+	Global global
+}
+
+// global struct of CloudConfig which is currently not inialized.
+type global struct {
+	Zone                        string `ini:"Zone,omitempty"`
+	VPC                         string `ini:"VPC,omitempty"`
+	SubnetID                    string `ini:"SubnetID,omitempty"`
+	RouteTableID                string `ini:"RouteTableID,omitempty"`
+	RoleARN                     string `ini:"RoleARN,omitempty"`
+	KubernetesClusterTag        string `ini:"KubernetesClusterTag,omitempty"`
+	KubernetesClusterID         string `ini:"KubernetesClusterID,omitempty"`
+	DisableSecurityGroupIngress bool   `ini:"DisableSecurityGroupIngress,omitempty"`
+	ElbSecurityGroup            string `ini:"ElbSecurityGroup,omitempty"`
+	DisableStrictZoneCheck      bool   `ini:"DisableStrictZoneCheck,omitempty"`
+}
+
+// serviceOverride struct used for AWS service endpoint override.
+type serviceOverride struct {
+	Service       string `ini:"Service"`
+	Region        string `ini:"Region"`
+	URL           string `ini:"URL"`
+	SigningRegion string `ini:"SigningRegion,omitempty"`
+	SigningMethod string `ini:"SigningMethod,omitempty"`
+	SigningName   string `ini:"SigningName,omitempty"`
+}
+
+// CloudProviderConfig builds the cloud provider config and reflects to an ini file.
+func CloudProviderConfig(params *awstypes.Platform) (string, error) {
```

Comment:
can we come up with a more descriptive name than `params`? You might also consider passing the overrides and region as separate params

Reply:
Renamed the variable to 'awsPlatform' instead.